src/main/steps/screen-questions/marriage-certificate/content.ts

```diff
@@ -51,10 +59,4 @@ export const marriageCertificateForm: FormContent = {
   },
 };
 
-const common = {
-  form: marriageCertificateForm,
-};
-
-export const marriageCertificateContent = { en, cy, common };
-
-export type MarriageCertificateForm = FormBody<typeof marriageCertificateForm>;
+export type MarriageCertificateForm = FormBody<typeof form>;
```

Comment:
is this still being used as now the postcontroller for it is gone

Reply:
👍 Good spot, I've removed this in ad6f948.